layers/meta-balena-beaglebone/conf/layer.conf

```diff
@@ -24,6 +24,14 @@ PREFERRED_PROVIDER_u-boot:beagleboard-xm = "u-boot"
 
 MACHINE_FEATURES:remove:beaglebone-ai64 = "efi"
 
+PREFERRED_PROVIDER_virtual/bootloader:beaglebone-ai64 = "u-boot-bb.org"
+PREFERRED_PROVIDER_u-boot:beaglebone-ai64 = "u-boot-bb.org"
```

Comment:
I think this is redundant

Reply:
I think the same *but* I am just following `meta-ti` here:

- [beaglebone-ai64.conf](https://git.yoctoproject.org/meta-ti/tree/meta-ti-bsp/conf/machine/beaglebone-ai64.conf?h=kirkstone) requires `conf/machine/include/j721e.inc`
- [conf/machine/include/j721e.inc](https://git.yoctoproject.org/meta-ti/tree/meta-ti-bsp/conf/machine/include/j721e.inc?h=kirkstone) requires `conf/machine/include/k3.inc`
- [conf/machine/include/k3.inc](https://git.yoctoproject.org/meta-ti/tree/meta-ti-bsp/conf/machine/include/k3.inc?h=kirkstone) defines it in this way.

If you want I can start removing options and see what happens... but I guess the people form `meta-ti` had a reason for it.